app/src/main/java/com/simplemobiletools/calculator/activities/SettingsActivity.kt

```diff
@@ -16,25 +13,22 @@ class SettingsActivity : SimpleActivity() {
         private lateinit var mConfig: Config
     }
 
-    @BindView(R.id.settings_dark_theme) lateinit var mDarkThemeSwitch: SwitchCompat
-
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_settings)
         mConfig = Config.newInstance(applicationContext)
-        ButterKnife.bind(this)
+
+        settings_dark_theme_holder.setOnClickListener {
```

Comment:
there is no dark theme anymore, it has been replaced with CustomizationActivity

Reply:
The Config.kt is not needed then, can I remove it?

Reply:
Removed in https://github.com/SimpleMobileTools/Simple-Calculator/pull/40/commits/8a8b2d23698f702e9e36d3046b1ca916a914d9ec, so you can see where it was used.